Sources/Models/LanguageModel.swift

```diff
@@ -129,6 +129,7 @@ extension LanguageModel {
         static let valueCache = "valueCache"
         // Output keys
         static let logits = "logits"
+        // swift-format-ignore: DontRepeatTypeInStaticProperties
```

Comment:
Lint check failing despite disabling rule globally in `.swift-format` and using this `swift-format-ignore:` directive...